scripts/pylib/twister/twisterlib/testinstance.py

```diff
@@ -278,11 +278,12 @@ def get_elf_file(self) -> str:
 
         fns = glob.glob(os.path.join(build_dir, "zephyr", "*.elf"))
         fns.extend(glob.glob(os.path.join(build_dir, "zephyr", "*.exe")))
+        fns.extend(glob.glob(os.path.join(build_dir, "testbinary")))
         blocklist = [
                 'remapped', # used for xtensa plaforms
                 'zefi', # EFI for Zephyr
                 '_pre' ]
-        fns = [x for x in fns if not any(bad in os.path.split(x)[-1] for bad in blocklist)]
+        fns = [x for x in fns if not any(bad in os.path.basename(x) for bad in blocklist)]
```

Comment:
Can you mention what this fix is for? On the face of it in looks like 'bad' iterates through the characters in the basename. Perhaps a comment would help

Reply:
This is just a style fix; the last (second) element of os.path.split(x) is equivalent to os.path.basename(x). It's checking to see if any of the strings in the blocklist are substrings of the ELF file names.

Reply:
Ah yes I see. Can you mention the style fix in your commit message?

Reply:
Thanks!